jpegenc/subblocks/RLE/RLECore/entropycoder.py

```diff
@@ -0,0 +1,38 @@
+""" This module takes a input and returns amplitude of the input and number of bits required to store the input """
```

Comment:
Should wrap this doc string, max length typically suggested for docstrings is 72